test/Libraries/Microsoft.Extensions.Diagnostics.Testing.Tests/Logging/FakeLoggerTests.cs

```diff
@@ -6,15 +6,36 @@
 using System.Globalization;
 using System.Linq;
 using System.Threading;
-using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Logging.Testing;
 using Microsoft.Extensions.Time.Testing;
 using Xunit;
 
 namespace Microsoft.Extensions.Logging.Testing.Test.Logging;
 
-public class FakeLoggerTests
+public partial class FakeLoggerTests
```

Comment:
I may be missing something obvious here, but I can't seem to understand what is the link between this and the rest of the PR.

Reply:
Hmmm, that's from a different PR. I don't know how it slipped into this one.

Reply:
Gone.